integration-test-sdk-net80/SearchingResourcesTest.cs

```diff
@@ -18,6 +19,8 @@ public void TestSearchingResources()
             AddValuesToSheet(smartsheet, sheetId, query);
             // this very rarely shows up in 5 seconds
             System.Threading.Thread.Sleep(5000);
+
+            // FIXME: this 500's out "{\n  \"errorCode\" : 4000,\n  \"message\" : \"An unexpected error has occurred. Please contact the Support team at https://help.smartsheet.com/contact for assistance.\",\n  \"refId\" : \"sn4akh\",\n  \"internalStackTrace\" : \"com.navigo.system.core.CriticalException: error while calling the Restmethod:com.navigo.smartsheet.rest.service.v2.resources.SearchResource#topLevelSearch\\nRetry count: 0\\n\\tat com.navigo.smartsheet.rest.framework.v2.MethodInvoker.rethrowIfNotRetriable(MethodInvoker.java:352)\\n\\tat com.navigo.smartsheet.rest.framework.v2.MethodInvoker.invokeWithRetry(MethodInvoker.java:278)\\n\\tat com.navigo.smartsheet.rest.framework.v2.MethodInvoker.lambda$executeRequest$1(MethodInvoker.java:196)\\n\\tat com.navigo.smartsheet.rest.framework.v2.filter.ApiFilterChain.executeNext(ApiFilterChain.java:26)\\n\\tat com.navigo.smartsheet.rest.framework.v2.config.smartsheet.BIMetadataFilter.execute(BIMetadataFilter.java:34)\\n\\tat com.navigo.smartsheet.rest.framework.v2.filter.ApiFilterChain.executeNext...
```

Comment:
What do you mean this 500's out? Like we are leaking a stack trace?

Reply:
Search was failing when I was testing in step box, but I pointed this to a prod account with a new API key and it works. I'm removing the `[Ignore]` tag.

Reply:
Sounds good!